packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -114,6 +114,21 @@ function startedInDebugMode(): boolean {
   return false;
 }
 
+// See https://github.com/Microsoft/vscode-languageserver-node/issues/105
+export function code2ProtocolConverter(value: vscode.Uri) {
+  if (/^win32/.test(process.platform)) {
```

Comment:
We use win32 and not win32 or win64 because that is how the process.platform value is defined. See https://nodejs.org/api/process.html#process_process_platform